news/20200128.misc

```diff
@@ -0,0 +1 @@
+Fixing git issues
```

Comment:
Is it worth having `news` files right now? Wouldn't we just do `Initial release` when this is up and running as a whole?

Reply:
I can remove them